tests/private-routes.test.ts

```diff
@@ -0,0 +1,41 @@
+import { createServer } from "../src/server";
+import { accessToken } from "./helpers";
+
+describe("private routes", () => {
```

Comment:
it'd be good to have a test that it's rejected when an expired token is present

this would be useful downstream too, if we're testing against this server – checking our code can cope with sending an expired token, refreshing it's token and resending

Reply:
I tried to write such a test, which should've been straightforward, but then I ran into this: https://giters.com/facebook/jest/issues/12056. Going to leave it for now but would like to come back to it.

Reply:
(there is a workaround there, but I don't want to do hideous jest workarounds this early in a project's life 😬 )